docs/source/api/recipes/recipes.ipynb

```diff
@@ -10,7 +10,7 @@
   },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #52.    pos = xgi.pca_transform(xgi.pairwise_spring_layout(H, seed=3))
</pre><p>Actually pos is redefined here - was it used in the meantime? Do we need two different pos variables?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/xgi-org/xgi/pull/535/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='54'/>

Reply:
You're right --- I deleted the duplicate